### PR TITLE
`General`: Add new contributors to about us page

### PR DESCRIPTION
**Pull request by MaximilianAnzinger**
#### General

- [x] This is a small issue that I tested locally and was confirmed by another developer on a test server.
- [x] I chose a title conforming to the [naming conventions for pull requests](https://ls1intum.github.io/Artemis/dev/development-process/#naming-conventions-for-github-pull-requests).

### Motivation and Context
Adds new contributors to the about us page.

### How to add yourself
1. Checkout the `chore/about-us` branch locally
2. Add yourself to this file: `src/main/resources/public/content/about-us.json`
 - Use `fullName` for your name
 - Use `sortBy` if the automatic sorting does not detect your last name correctly (if your name consists of mulitple words)
3. Add your image to the `public/images/about` directory and link that image using the `photoDirectory` key. Make sure to use a square, otherwise crop your image before uploading it.
4. *Make sure the file has at most 100KB*

### Steps for Testing
View the about us page and check that the new contributors are visible and sorted.

### CODE REVIEW

**Review by vinceclifford — DISMISSED** [review]
Works as expected

**Review by rstief — APPROVED** [review]
Tested on ts3 all new additions are there and page looks good.

File: src/main/resources/public/images/about/Maximilian_Anzinger.jpg
Comment (krusche) [inline src/main/resources/public/images/about/Maximilian_Anzinger.jpg:1]: This file is too large. Please provide a smaller version that is at most 100KB

File: src/main/resources/public/images/about/Laurenz_Blumentritt.jpg
Comment (krusche) [inline src/main/resources/public/images/about/Laurenz_Blumentritt.jpg:1]: This file is too large. Please provide a smaller version that is at most 100KB

File: src/main/resources/public/images/about/Valentin_Boehm.jpg
Comment (krusche) [inline src/main/resources/public/images/about/Valentin_Boehm.jpg:1]: This file is too large. Please provide a smaller version that is at most 100KB

File: src/main/resources/public/images/about/Andreas_Resch.png
Comment (krusche) [inline src/main/resources/public/images/about/Andreas_Resch.png:1]: This file is too large. Please provide a smaller version that is at most 100KB

**Review by krusche — CHANGES_REQUESTED** [review]
some images are too large, please provide smaller versions

**Review by rstief — APPROVED** [review]

**Review by JohannesStoehr — APPROVED** [review]
Updated links work and changes overall look good!

**Review by vinceclifford — APPROVED** [review]
LGTM, tested on TS3

**Review by Predixx — APPROVED** [review]
Tested on ts3 with Vincent Clifford. Images all look great and are displayed correctly

**Pull request merged**